ocp_diagnosis.sh

```diff
@@ -125,6 +126,23 @@ if [[ "$PROMETHEUS_CAPTURE" == "true" ]]; then
 	fi
 fi
 
-if [[ "$OPENSHIFT_MUST_GATHER" == "true" ]]; then
-	must_gather
+# if [[ "$OPENSHIFT_MUST_GATHER" == "true"]]; then
+# 	must_gather
+# 	curl $DATA_SERVER --form file@"$OUTPUT_DIR/must-gather-$ts.tar.xz"
+# fi
+
+# validate data server string is not empty
+if [[ -n "$DATA_SERVER" ]]; then
```

Comment:
we can replace this with
if [[ STORAGE_MODE == data_server]]
  then do stuff
fi
and remove the check inside

Reply:
No plans to check `$DATA_SERVER`?